gcloud-java-examples/README.md

```diff
@@ -39,6 +39,17 @@ To run examples from your command line:
 
 4. Run an example using Maven from command line.
 
+  Here's an example run of `BigQueryExample`.
+
+  Before running the example, go to the [Google Developers Console][developers-console] to ensure that BigQuery API is enabled.
```

Comment:
I think it'd be good to add a note here that mentions that the user should:
* create a test csv file with some strings, one string per line.
* upload the csv file Cloud Storage (using the Developers Console)